css/types/color.json

```diff
@@ -281,6 +281,56 @@
             }
           }
         },
+        "color-contrast": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-contrast()",
+            "spec_url": "https://www.w3.org/TR/css-color-5/#colorcontrast",
+            "description": "<code>color-contrast()</code>",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": {
+                "version_added": false
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": "15"
```

Comment:
needs enabled with the flag `CSSColorContrastEnabled` which is in the dropdown menu.. todo: need to go find the way to reference this.

Reply:
Thanks for getting started on this, @argyleink!

This can be done with the `flags` key. See https://github.com/mdn/browser-compat-data/blob/main/css/properties/scroll-behavior.json#L33 for an example and [the schema docs](https://github.com/mdn/browser-compat-data/blob/main/schemas/compat-data-schema.md#flags) for details. Also note, we usually use the string from the actual UI, rather than name of the preference in source.

I'm not sure that Safari for iOS supports flags, so depending on the menu options in iOS, it might be that it's still `false`.

Reply:
thanks for the pointers! items have been attended to 👍🏻